app/services/finance/ecf/participant_aggregator.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Finance
+  module ECF
+    class ParticipantAggregator < Finance::ParticipantAggregator
+      def self.aggregation_types
+        {
+          started: {
+            # not_yet_included: :not_eligible_for_lead_provider,
```

Comment:
✂️

Reply:
yea, i want to leave this to fix another time. it no longer makes sense in my opinion but would also need removing from the view